### ISSUE TITLE
First dialog field should get the keyboard focus

### ISSUE THREAD

**Issue by rillig** [manolo/gwt-polymer-todo#12]
In the Widgets TodoList application, when I click on the red [+] button in the lower right corner, a dialog opens. I'd like to start typing immediately, but there is no text cursor.

When opening the dialog, the keyboard focus should be on the Title text field.
